src/config.tmpl.h

```diff
@@ -0,0 +1,20 @@
+/* **************** OSW LABS ****************** */
+/**
+ * @license
+ * Copyright OSW Labs. All Rights Reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+/* *************** AgroLab IoT **************** 
+ * Authors:  
+  - Sergio Morcuende (https://github.com/smorcuend)
+  - Rabindranath Andujar (https://github.com/ndujar)
+ * Source Code: https://github.com/OSWHackerspace/AgrolabIoT 
+ * Community: https://github.com/osweekends
+*/
+#define USERNAME ""
```

Comment:
We could add here indications for students as to what these variables are for.
/* ************************* AgroLab IoT **********************************
USERNAME: Put here your user name in the IoT platform thinger.io
DEVICE_ID: Put here the ID of the ESP8266 device in the thinger.io platform
DEVICE_CREDENTIAL: Put here the ID of the ESP8266 device in the thinger.io platform
*/